modules/context/repo.go

```diff
@@ -1097,3 +1113,54 @@ func (ctx *Context) IssueTemplatesErrorsFromDefaultBranch() ([]*api.IssueTemplat
 	}
 	return issueTemplates, invalidFiles
 }
+
+func ExtractIssueConfigFromYaml(configContent []byte) (api.IssueConfig, error) {
+	config := api.IssueConfig{
+		BlankIssuesEnabled: true,
```

Comment:
Why is the default true?

Reply:
The blank Issues should be enabled, if nothing other specified